tests/test_indexer.py

```diff
@@ -182,7 +182,7 @@ def test_indexed_file_access_error(self):
             self.process_new_indexable_object(sample_event, logger)
         self.assertRegex(log_monitor.output[0],
                          f"WARNING:.*:In bundle .* the file \"{inaccesssible_filename}\" is marked for indexing"
-                         " yet could not be accessed. This file will not be indexed. Exception:")
+                         " yet could not be accessed. This file will not be indexed. Exception: .* File blob key:")
```

Comment:
super nitpicky: should be:

`Exception: .*, File blob key: `

Reply:
Thanks for the suggestion. Done.